### DISCUSSION TITLE
Game labels and tags

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#695] [Ideas]
Basically elements of game information, but used as a tool to browse through games, by clicking on them.
Currently we have:
- Developer
- Publisher
- Genre
- First released

They should be supplemented at least by two more fields:
- Franchise
- Multiplayer (Yes/No)

Additionaly First released could be split by date (clicking on the month) and by year (clicking on the year), and Franchise could additionally consist game series (collapsable or separated).
After clicking and displaying games by Developer for example, they should be grouped by system from which the browsing was initiated, but on the same screen there should be an option "display all game by Developer", which would show games for all systems.

Optionally we could add another 2 labels based on information from GameFAQs:
- Difficulty
- Length

**Comment by ScottFromDerby** [comment]
Is multiplayer always going to be yes or no? Might only be something to consider for more modern non-retro games...

**Comment by ScottFromDerby** [comment]
I.e. mp 2-4 players, 2 players coop, 2 players over internet

**Comment by julkasalsa** [comment]
Yeah absolutely, it could be more specified.
